src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -319,6 +319,31 @@ public void updateInfo(BoxFile.Info info) {
     }
 
     /**
+     * Checks if the file can be successfully uploaded by using the preflight check.
+     * @param  name        the name to give the uploaded file.
```

Comment:
I think it might be worth mentioning that the name can be null in order to use existing name.